js/components/graph/GraphDropdown.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default class GraphDropdown extends React.Component{
+
+	generateDropdownItems(column){
```

Comment:
After reflecting on this a bit, I don't think Courseography should be imposing categories on the graphs. Revert this change to the two-column layout you had before, and do not use any headings. This should greatly simplify this component.